spanner/client.go

```diff
@@ -1504,3 +1526,13 @@ func parseServerTimingHeader(md metadata.MD) map[string]time.Duration {
 	}
 	return metrics
 }
+
+// enableClientOptionsLog returns true if env doesn't exist or false
+func enableClientOptionsLog() bool {
+	if disableLogString, found := os.LookupEnv("GOOGLE_CLOUD_SPANNER_DISABLE_CLIENT_OPTIONS_LOGS"); found {
+		if disableLog, err := strconv.ParseBool(disableLogString); err == nil {
+			return !disableLog
+		}
+	}
+	return true
+}
```

Comment:
nit, can we rename these a bit:


```suggestion
// enableLogClientOptions returns true if the environment variable for this doesn't exist or is set to false
func enableLogClientOptions() bool {
	if disableLogString, found := os.LookupEnv("GOOGLE_CLOUD_SPANNER_DISABLE_LOG_CLIENT_OPTIONS"); found {
		if disableLog, err := strconv.ParseBool(disableLogString); err == nil {
			return !disableLog
		}
	}
	return true
}
```